### PR TITLE
Update devDependencies and peerDependencies to accepts Backbone 1.3.3. Sync lib with src.

### LINKED ISSUE #75: Update peerDependencies
We should update the peerDependencies for babysitter for the Mn v3 release.  I think we can get away with possibly setting less restrictions on this library that maybe we would for Mn.

Possibly just:
```
"peerDependencies": {
    "backbone": "^1.3.3",
    "underscore": "^1.8.3"
  }
```
Since they're peer Deps anyone could still install older versions, but it would just complain on npm i.

### PR DESCRIPTION
**Pull request by rafde**
* peerDependency accepts Backbone ^1.3.3 and underscore ^1.8.3
* devDependency accepts Backbone >=1.3.3 but <1.4.0
  since we don't know if Backbone will introduce a breaking changes

* build to sync lib with source

closes #75 

Please create release tag for when this PR is merged.

### CODE REVIEW

**Comment by paulfalgout** [comment]
Ah.. there's stuff on the major branch  @ahumphreys87 this should be part of the v1 stuff

File: package.json [hunk 1]
```diff
@@ -38,11 +38,11 @@
...
   "github": "https://github.com/marionettejs/backbone.babysitter",
   "peerDependencies": {
-    "backbone": ">=0.9.9 <=1.3.2",
-    "underscore": ">=1.4.0 <=1.8.3"
+    "backbone": "^1.3.3",
+    "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "backbone": ">=0.9.9 <=1.3.2",
+    "backbone": ">=1.3.3 <1.4.0",
```
Comment (rafde) [inline package.json]: I think I can switch this to `>=0.9.9` since it really only uses `Backbone` as a namespace object.

File: package.json [hunk 2]
```diff
@@ -38,11 +38,11 @@
   },
   "github": "https://github.com/marionettejs/backbone.babysitter",
   "peerDependencies": {
-    "backbone": ">=0.9.9 <=1.3.2",
-    "underscore": ">=1.4.0 <=1.8.3"
+    "backbone": "^1.3.3",
```
Comment (rafde) [inline package.json]: https://github.com/marionettejs/backbone.babysitter/pull/76/files#r71965325

**Comment by rafde** [comment]
@paulfalgout @ahumphreys87 willing to take a look?

**Comment by ahumphreys87** [comment]
👍

**Pull request merged**